### PR TITLE
feat: Add conditions to automate the installation for CRDs.

### PR DESCRIPTION
**Pull request by tao12345666333**
#### What this PR does / why we need it:
Use this PR as a replacement for https://github.com/Kong/charts/pull/1023

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes https://github.com/Kong/gateway-operator/issues/1323

#### Special notes for your reviewer:

I accidentally covered https://github.com/Kong/charts/pull/1023 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] PR is based off the current tip of the `main` branch.
- [ ] Changes are documented under the "Unreleased" header in CHANGELOG.md
- [ ] New or modified sections of values.yaml are documented in the README.md
- [ ] Commits follow the [Kong commit message guidelines](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#commit-message-format)

### CODE REVIEW

File: charts/gateway-operator/charts/gwapi-standard-crds/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# GWAPI standard CRDs subchart
+
+This sub-chart contains Gateway API standard CRDs, allowing users to control whether or not to install them themselves.
```
Comment (randmonkey) [inline charts/gateway-operator/charts/gwapi-standard-crds/README.md]:
```suggestion
This sub-chart contains Gateway API standard CRDs, allowing users to control whether to install the CRDs.
```

File: charts/gateway-operator/charts/gwapi-experimental-crds/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# GWAPI experimental CRDs subchart
+
+This sub-chart contains Gateway API experimental CRDs, allowing users to control whether or not to install them themselves.
```
Comment (randmonkey) [inline charts/gateway-operator/charts/gwapi-experimental-crds/README.md]: Ditto here.

File: charts/gateway-operator/charts/kic-crds/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# KIC CRDs subchart
+
+This sub-chart contains KIC's CRDs, allowing users to control whether or not to install them themselves.
```
Comment (randmonkey) [inline charts/gateway-operator/charts/kic-crds/README.md]:
```suggestion
This sub-chart contains Kong Ingress Controller (KIC)'s CRDs, allowing users to control whether to install them.
```

File: charts/gateway-operator/charts/gwapi-experimental-crds/README.md (hunk 2 above)
Comment (tao12345666333) [inline charts/gateway-operator/charts/gwapi-experimental-crds/README.md]:
```suggestion
This sub-chart contains Gateway API experimental CRDs, allowing users to control whether to install the CRDs.
```

**Review by randmonkey — APPROVED** [review]

**Pull request merged**